### PR TITLE
Restore cmake system/bundled external libs option

### PR DESCRIPTION
**Pull request by alemuntoni**

### CODE REVIEW

**Comment by alemuntoni** [comment]
@rpavlik are these options ok for making easier debian builds?

File: src/external/external.cmake [hunk 1]
```diff
@@ -24,22 +28,28 @@ add_subdirectory(${MUPARSER_DIR})
...
-set(SSYNTH_DIR ${EXTERNAL_DIR}/structuresynth-1.5)
-add_subdirectory(${SSYNTH_DIR})
+if (ALLOW_BUNDLED_LIBS)
+	# structure-synth - optional, for filter_ssynth
+	set(SSYNTH_DIR ${EXTERNAL_DIR}/structuresynth-1.5)
+	add_subdirectory(${SSYNTH_DIR})
+endif()
 
 # qhull - optional, for filter_qhull
 set(QHULL_DIR ${EXTERNAL_DIR}/qhull-2003.1)
```
Comment (rpavlik) [inline src/external/external.cmake:38]: so Debian has a package for qhull, but not e.g. newuoa, etc. It's not that we want to exclude the qhull plugin, we just want to be able to use the system-provided qhull, which the previous build system (and maybe this one?) did search for and try to use.

Comment (alemuntoni) [reply src/external/external.cmake:38]: Yep, this build system will first search for system libraries (if they are enabled) and, if not provided, will fall back to the bundled ones (if they are enabled).

Comment (alemuntoni) [reply src/external/external.cmake:38]: But the check is done inside the qhull directory. But it could be moved here if it is better

Comment (rpavlik) [reply src/external/external.cmake:38]: yeah, we drop the whole qhull directory since that's the easiest thing. See #847

**Pull request closed without merging**